### DISCUSSION TITLE
post/paytalks-with-krew-fe/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#51] [Announcements]
# post/paytalks-with-krew-fe/

Paytalks with KREW: FE개발에 진심인 분들을 초대합니다. 여러분의 열정과 고민, 카카오페이 크루와 함께 나눠요.

https://tech.kakaopay.com/post/paytalks-with-krew-fe/

**Comment by yrnana** [comment]
많은 인사이트 얻고갑니다!

**Comment by haneulcho** [comment]
요즘 리드를 계속 해야하는지, 그냥 개발자일 때가 편했지 않나 하는 현타가 왔는데... 읽으면서 공감도 되고 느끼는 점도 많았어요. 잘 읽었습니다! ☺️
